code/modules/reagents/reagent_containers/food/condiment.dm

```diff
@@ -580,3 +580,58 @@
 /obj/item/weapon/reagent_containers/food/condiment/chillwax/New()
 	..()
 	reagents.add_reagent(CHILLWAX, 50)
+
+//////////////////////////////////////////////////////////////////////////
+
+/obj/item/weapon/reagent_containers/food/condiment/small
+	possible_transfer_amounts = list(1, 5)
+	amount_per_transfer_from_this = 1
+	var/trash_type
+
+/obj/item/weapon/reagent_containers/food/condiment/small/on_reagent_change()
+	if(reagents.reagent_list.len <= 0)
+		if (trash_type)
+			var/obj/item/trash/trash = new trash_type(loc)
+			if (ismob(loc))
+				var/mob/M = loc
+				if (M.get_active_hand() == src)
```

Comment:
We can do this more flexibly
`var/hand_index = M.is_holding_item(src)
[drop the item code here]
M.put_in_hand(hand_index, src)`

No matter how many hands, even a Martian, this will place in the correct hand